frontend/packages/ux-editor/src/containers/DesignView/FormTree/FormItem/FormItemTitle/FormItemTitle.module.css

```diff
@@ -4,6 +4,7 @@
 
 .label {
   flex: 1;
+  width: 80%;
```

Comment:
Should we add a comment on why we need 80% here? It's done because we need space for the delete/trash-icon. 😊